bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/listener/DoorbirdUdpListener.java

```diff
@@ -97,7 +98,6 @@ private void receivePackets() {
                 // Nothing to do on socket timeout
             } catch (IOException e) {
                 logger.debug("Listener got IOException waiting for datagram: {}", e.getMessage());
-                socket = null;
```

Comment:
Why was it necessary to remove this?

Reply:
Refactored this piece of code and was not satisfied with the result so i reverted and overlooked this.
Reverted this line.